### PR TITLE
[VarDumper] consistent signature of getDump() in class + trait

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #16126 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Component/VarDumper/Test/VarDumperTestCase.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ public function assertDumpMatchesFormat($dump, $data, $message = '')
         $this->assertStringMatchesFormat(rtrim($dump), $this->getDump($data), $message);
     }
 
-    private function getDump($data)
+    public function getDump($data)
```
Comment (xabbuh) [inline src/Symfony/Component/VarDumper/Test/VarDumperTestCase.php]: @nicolas-grekas Wouldn't it be enough to have the method `protected` in both the class and the trait?

Comment (nicolas-grekas) [inline src/Symfony/Component/VarDumper/Test/VarDumperTestCase.php]: I think yes

Comment (xabbuh) [inline src/Symfony/Component/VarDumper/Test/VarDumperTestCase.php]: alright, I changed both methods

**Comment by nicolas-grekas** [comment]
:+1:

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**